src/lib-gantry.sh

```diff
@@ -400,7 +400,17 @@ _report_services() {
   local TITLE BODY
   TITLE="[${STACK}] ${NUM_UPDATED} services updated ${NUM_FAILED} failed${ERROR_STRING}"
   BODY=$(echo -e "${UPDATED_MSG}\n${FAILED_MSG}\n${ERROR_MSG}")
-  _send_notification "${TYPE}" "${TITLE}" "${BODY}"
+
+  case "${GANTRY_NOTIFICATION_CONDITION:-all}" in
```

Comment:
Assign global variable to a local one like line 276.
This avoids accidentally changing the global variable, or the global variable get different default values.